cmd/imagec/docker.go

```diff
@@ -55,11 +61,14 @@ type Manifest struct {
 
 // V1Compatibility represents some parts of V1Compatibility
 type V1Compatibility struct {
-	ID        string    `json:"id"`
-	Parent    string    `json:"parent,omitempty"`
-	Comment   string    `json:"comment,omitempty"`
-	Created   time.Time `json:"created"`
-	Container string    `json:"container,omitempty"`
+	docker.V1Image
```

Comment:
Hmm now that you embedded the whole V1Image I guess the comment is no longer true (it's not some parts of V1Compatibility, it's V1 now :P) and maybe we can rename this struct to something more meaningful .